src/components/SectionWarranty/styles.js

```diff
@@ -0,0 +1,32 @@
+import styled from "@emotion/styled";
+
+const style = (Component) => styled(Component)`
+  .titulo-strong {
+    font-family: Lato;
+    font-size: 32px;
+    font-style: normal;
```

Comment:
There's no need to set the font-style to normal. That's the default value.

Reply:
Fixed in last commit.